pkg/reconciler/configuration/configuration.go

```diff
@@ -223,6 +224,7 @@ func CheckNameAvailability(config *v1.Configuration, lister listers.RevisionList
 	} else if !metav1.IsControlledBy(rev, config) {
 		// If the revision isn't controller by this configuration, then
 		// do not use it.
+		logger.Warnf("The revision is not controlled by this configuration. Rev.meta: %#v", rev.GetOwnerReferences())
```

Comment:
Would it be helpful to also log the rev name here?

Reply:
Thx, updated